lib/gems/pending/util/extensions/miq-benchmark.rb

```diff
@@ -29,6 +29,10 @@ def self.realtime_block(key, &block)
     begin
       ret = realtime_store(hash, key, &block)
       return ret, hash
+    rescue
+      # Don't let timings be lost on exception when there is nobody else to pick them up.
+      $log.info("Exception in realtime_block #{key.inspect} - Timings: #{hash.inspect}") if outermost
```

Comment:
@cben I agree.  

* Creating an instance that you pass a logger seems like a hassle.

* Using a class accessor on Benchmark to set a logger might work but then the caller knows about benchmark... I guess.  It might work?  See [here](https://github.com/ManageIQ/manageiq/blob/aad44397f41df5a2dbb1b4691307887518b3c2f8/lib/vmdb/loggers.rb#L63) where we create the loggers and setup the external loggers.

* stderr should work too.  Try that first and see if the workers log it.  I think that's the simplest approach.  We should be keeping stderr.

```
app/models/miq_worker.rb:    pid = Kernel.spawn(self.class.build_command_line(guid), [:out, :err] => [Rails.root.join("log", "evm.log"), "a"])
lib/tasks/evm_application.rb:    pid = Kernel.spawn(env_options, command_line, :pgroup => true, [:out, :err] => [Rails.root.join("log/evm.log"), "a"])
```